Add enterprise image signing verification docs (#6730)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/index.md

Starting in version 3.2, {{site.base_gateway}} can be configured to support configuring new data planes in the event of a control plane outage. For more information, read the [How to Manage New Data Planes during Control Plane Outages](/gateway/latest/kong-enterprise/cp-outage-handling/) documentation, or the [Control Plane Outage Management FAQ](/gateway/latest/kong-enterprise/cp-outage-handling-faq/).

{% endif_version %}

## Docker container image signing

> Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are signed, and can be verified using `cosign` with signatures published to a Docker Hub repository. Read the [Verify signatures for Signed Kong Images](/gateway/latest/kong-enterprise/signing-verify/) documentation to learn more.

Reviewer: ```suggestion
{% if_version gte:3.5.x %}
Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are signed, and can be verified using `cosign` with signatures published to a Docker Hub repository. Read the [Verify signatures for Signed Kong Images](/gateway/{{ page.release }}/kong-enterprise/signing-verify/) documentation to learn more.
{% endif_version %}
```

Author: I don't think I understand why we need conditionality here. Written without version conditions, this is a pretty compelling wrap sheet of goodies you get by upgrading.

I'm fine either way really.

Reviewer 2: If we don't add conditional tags here, this link has to go to "latest" and never a specific version, as it'll break in anything before 3.5.

While you _could_ argue that this page specifically is meant as a sort of marketing/advertisement blurb page anyway, we just haven't historically treated it that way. It's more of a "here are the things available in the version that you're running" rather than "here are all the things you could get if you upgraded". 

So - yes, we should probably revisit the purpose and approach of this page, but I don't think that we should change the approach for just one feature.

Author: Thanks for the thoughtful explanation, that makes sense.

Reviewer: Exactly that - thanks for clarifying Lena

Starting in version 3.2, {{site.base_gateway}} can be configured to support configuring new data planes in the event of a control plane outage. For more information, read the [How to Manage New Data Planes during Control Plane Outages](/gateway/latest/kong-enterprise/cp-outage-handling/) documentation, or the [Control Plane Outage Management FAQ](/gateway/latest/kong-enterprise/cp-outage-handling-faq/).

{% endif_version %}

## Docker container image signing

> Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are signed, and can be verified using `cosign` with signatures published to a Docker Hub repository. Read the [Verify signatures for Signed Kong Images](/gateway/latest/kong-enterprise/signing-verify/) documentation to learn more.

Reviewer 2: ```suggestion
Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are signed, and can be verified using `cosign` with signatures published to a Docker Hub repository. Read the [Verify Signatures for Signed Kong Images](/gateway/latest/kong-enterprise/signing-verify/) documentation to learn more.
```

Starting in version 3.2, {{site.base_gateway}} can be configured to support configuring new data planes in the event of a control plane outage. For more information, read the [How to Manage New Data Planes during Control Plane Outages](/gateway/latest/kong-enterprise/cp-outage-handling/) documentation, or the [Control Plane Outage Management FAQ](/gateway/latest/kong-enterprise/cp-outage-handling-faq/).

{% endif_version %}

> 
> ## Docker container image signing
> 
> {% if_version gte:3.5.x %}

Reviewer 2: ```suggestion

{% if_version gte:3.5.x %}

## Docker container image signing

```
